### DISCUSSION TITLE
Fixed Scipy.libs FileNotFoundError Issue

### DISCUSSION THREAD

**Discussion by EuphoricPenguin** [voicepaw/so-vits-svc-fork#379] [General]
I thought I would post stating that using Python 3.8 within Anaconda resolved the issue I had initially tried to get help with in #180. Issues with LLM self-botting aside, which I still strongly recommend against, this is precisely what solved my issue. I noticed that the error printout noted something was deprecated in the more recent version of Scipy, so I thought to try an older version of Python. I'm not sure if this is exactly what was fixed by using an older version, but I have the training up and running now as a result. I thought I would share, just in case anyone else runs into this issue.

**Comment by lenover12** [comment]
I also had this problem, I created my virtual environment via python venv with an older version of python
but the next time I opened this program I remade the venv with python3 which is my most up to date version of python.
I had to change it back with python venv and then uninstall Scipy and reinstall it from the instructions
